tools/lsp/ui/components/property-widgets.slint

```diff
@@ -472,53 +480,20 @@ component StringWidget inherits VerticalLayout {
 
                     tr-context := ResettingLineEdit {
                         enabled: root.enabled && tr-cb.checked;
+                        default-text: root.tr-context-value;
                         edited(text) => {
+                            root.tr-context-value = text;
                             self.can-compile = root.tsb();
                         }
                         accepted(text) => {
+                            root.tr-context-value = text;
                             root.ssb();
                         }
+                        changed has-focus => {
+                            root.child-focus = self.has-focus;
+                        }
                     }
                 }
-
-                // Row {
```

Comment:
(you can also remove the comment about FIXME about plural, i think. There is nothing to fix right now.)

Reply:
I kept it since this is an issue for the property editor still.